admin/CR-vcdm/Approval_request.md

```diff
@@ -1,64 +1,90 @@
-(Draft for a CR Transition DI, cryptosuites, json schemas)
-
+(Draft for a CR transition of VCDM v2.0 and VC Specifications Directory)
 
 # Document title, URLs, estimated publication date
 
 - Verifiable Credentials Data Model v2.0
-  - https://www.w3.org/TR/2023/CR-vc-data-model-2.0-202312@@/
-  - @@ December 2023
+  - https://w3c.github.io/vc-data-model/CR/2023-12-21/
+  - 21 December 2023
   - Editors' draft: https://w3c.github.io/vc-data-model/
 
 # Abstract
 
-- https://www.w3.org/TR/2023/CR-vc-data-model-2.0-202312@@/#abstract
+- https://w3c.github.io/vc-data-model/CR/2023-12-21/#abstract
 
 # Status
 
-- https://www.w3.org/TR/2023/CR-vc-data-model-2.0-202312@@/#sotd
-  
+- https://w3c.github.io/vc-data-model/CR/2023-12-21/#sotd
+
 # Link to group's decision to request transition
 
 - https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-12-@^@#resolution@^^@
 
 # Changes
 
-- https://www.w3.org/TR/2023/CR-vc-data-model-2.0-202312@@/#revision-history
-- 
+- https://w3c.github.io/vc-data-model/CR/2023-12-21/#revision-history
+
 # Requirements satisfied
 
 Yes.
 
 # Dependencies met (or not)
 
+All normative dependencies are either RECs or IETF RFCs, except for:
+
+* Verifiable Credential Data Integrity, which entered CR in November 2023.
+* Securing Verifiable Credentials using JOSE and COSE, which is a WD and expected to enter CR in Q1 2024.
 
 # Wide Review
 
 Issues processed:
-- https://github.com/w3c/vc-data-model/issues?q=is%3Aissue+is%3Aclosed
-- 
+- https://github.com/w3c/vc-data-model/issues?q=is%3Aissue+is%3Aclosed+created%3A%3E2022-06-13
+
 PRs processed:
-- https://github.com/w3c/vc-data-model/pulls?q=is%3Apr+is%3Aclosed
-- 
+- https://github.com/w3c/vc-data-model/pulls?q=is%3Aissue+is%3Aclosed+created%3A%3E2022-06-13
+
 Horizontal reviews:
-* TAG: 
-* PING: 
-* Security: 
-* a11y: 
-* i18n: 
+* TAG: https://github.com/w3ctag/design-reviews/issues/860 (request timeout)
```

Comment:
They way I read https://github.com/w3ctag/design-reviews/issues/850#issuecomment-1664352922 it is pretty much looking as an approval; no substantial changes required. Also, I seem to remember that both TAG reviewers were at one of our F2F meetings in Sevilla.

I think it would be worth asking the reviewers to put an official mark into the issue whereby this is simply completed. That would look way better than a "request timeout".

Reply:
That TAG design review was for VC Data Integrity, not the core data model.

I have asked the reviewers to update the VCDM review request here:

https://github.com/w3ctag/design-reviews/issues/860#issuecomment-1821056346